packages/riipen-ui/src/utils/withClasses.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+
+const withClasses = () => Component => {
+  if (process.env.NODE_ENV !== "production") {
+    if (Component === undefined) {
+      throw new Error(
+        [
+          "You are calling withStyles(styles)(Component) with an undefined component.",
+          "You may have forgotten to import it."
+        ].join("\n")
+      );
+    }
+  }
```

Comment:
can we remove this? this will error when rendered anyways. And I think all components should get rendered as part of our doc site / demo?